apps/BasicExample/package.json

```diff
@@ -8,7 +8,10 @@
     "lint": "eslint .",
     "start": "react-native start",
     "test": "jest",
-    "postinstall": ""
+    "postinstall": "",
```

Comment:
```suggestion
```

Reply:
Well, I was sure that `yarn` disabled calling `pre` and `post` scripts on CI. Turns out that only the output is silenced so I guess we can get rid of this and remove explicitly calling `postinstall`. Done in [ca5aba3](https://github.com/software-mansion/react-native-gesture-handler/pull/3503/commits/ca5aba39acf2a1aeede917fc95a17f78246e3f91).